build.sh

```diff
@@ -1,8 +1,15 @@
 #!/usr/bin/env bash
 
+unameOut="$(uname -s)"
+case "${unameOut}" in
+    Linux*)     command=xdg-open;;
+    Darwin*)    command=open;;
+esac
+echo ${command}
+
 LANGUAGE=$1
```

Comment:
exit 1 if arg wasn't found? else the process carries on

Reply:
It's fine. Fails exactly where it should.